api/repositories/service_binding_repository.go

```diff
@@ -75,6 +77,12 @@ func (r ServiceBindingRecord) Relationships() map[string]string {
 	}
 }
 
+type Credentials map[string]any
```

Comment:
Is there any reason to alias `map[string]any` as `Credentials`? I.e. is the `Credentials` type needed?